ENG4000-Team-A/capstone-project
# Update model upon login and improve model spec

Create a new User object upon successful login, update the user model

To test
1. Run: python3 manage.py runserver
2. Run: ./ServerSocket.py
3. Run: ./ExternalClientSocket.py
4. Go to http://127.0.0.1:8000/login
5. Enter a valid username, password, and click login (username="chris354", password="greet984;d")
6. Go to http://127.0.0.1:8000/admin and log into your super user account
7. See the existing instances of user objects and you should see the new user created
